tools/js-optimizer.js

```diff
@@ -6649,22 +6649,28 @@ function emterpretify(ast) {
             if (type === ASM_INT) {
               opcode = 'ADD';
               tryNumSymmetrical();
-            } else if (type === ASM_DOUBLE) opcode = 'ADDD';
+            } else if (type === ASM_DOUBLE || type === ASM_FLOAT) {
```

Comment:
Similar verboseness in llvm http://llvm.org/docs/doxygen/html/classllvm_1_1Type.html#a40113570b0c52a18c34cabed8359e2c8